packages/components/src/RadioGroup/RadioGroup.css

```diff
@@ -12,6 +12,7 @@
 .label {
   display: inline-flex;
   align-items: center;
+  padding-bottom: 0.5rem;
```

Comment:
This should use one of our spacing units.

https://atlantis.getjobber.com/space